docs/deprecations.md

```diff
@@ -6,6 +6,18 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+<a name="AWS_FUNCTION_DESTINATIONS_MAXIMUM_EVENT_AGE"><div>&nbsp;</div></a>
+
+## AWS Lambda Function Destinations `maximumEventAge`
+
+Please use `maximumEventAge` instead of `destinations.maximumEventAge`. Support for `destinations.maximumEventAge` will be removed with v2.0.0
+
+<a name="AWS_FUNCTION_DESTINATIONS_MAXIMUM_RETRY_ATTEMPTS"><div>&nbsp;</div></a>
+
+## AWS Lambda Function Destinations `maximumRetryAttempts`
+
+Please use `maximumRetryAttempts` instead of `destinations.maximumRetryAttempts`. Support for `destinations.maximumRetryAttempts` will be removed with v2.0.0
+
```

Comment:
For simplicity let's use one deprecation, we can word it as:

_`maximumEventAge` and `maximumEventAge` should be defined directly at function level. Support for those settings on `destinations` level, will be removed with v2.0.0_

And code could probably be `AWS_FUNCTION_DESTINATIONS_ASYNC_CONFIG`

Reply:
Great idea :100: